test/test_init.py

```diff
@@ -376,13 +367,11 @@ def test_multilump(ctx_factory, dim):
                               spec_centers=centers, velocity=velocity,
                               spec_y0s=spec_y0s, spec_amplitudes=spec_amplitudes)
 
-    lump_soln = lump(nodes)
-    numcvspec = get_num_species(dim, lump_soln)
+    cv = lump(nodes)
+    numcvspec = get_num_species(dim, cv.join())
```

Comment:
Should we deprecate `get_num_species`? (OK to turn that into an issue.)

Reply:
What about making it internal-only?   Or would you like to make an `nspecies` property?